observer/fly.toml

```
@@ -7,3 +7,4 @@ primary_region = "cdg"
 
 [deploy]
   strategy = "rolling"
+  release_command = "npm run migrate"
```

I think this will run the root-level migrate script, which in turn runs migrate scripts in all sub-packages (stats, observer). If I am correct, then we can fix that as follows:

```suggestion
  release_command = "npm run migrate --workspace observer"
```